### PR TITLE
[zh] sync cluster-role-binding-v1 cluster-role-v1 role-binding-v1 role-v1

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
/approve

File: content/zh-cn/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ ClusterRoleBinding 引用 ClusterRole，但不包含它。
 - **roleRef** (RoleRef)，必需
 
   RoleRef 只能引用全局命名空间中的 ClusterRole。
-  如果无法解析 RoleRef，则 Authorizer 必定返回一个错误。
+  如果无法解析 RoleRef，则 Authorizer 必定返回一个错误。这个字段是不可变的。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md:72]:
```suggestion
  如果无法解析 RoleRef，则 Authorizer 必定返回一个错误。这个字段是不可变更的。
```

File: content/zh-cn/docs/reference/kubernetes-api/authorization-resources/role-binding-v1.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ RoleBinding 通过 Subjects 和所在的命名空间信息添加主体信息。
 - **roleRef** (RoleRef)，必需
   
   roleRef 可以引用当前命名空间中的 Role 或全局命名空间中的 ClusterRole。
-  如果无法解析 roleRef，则 Authorizer 必定返回一个错误。
+  如果无法解析 roleRef，则 Authorizer 必定返回一个错误。这个字段是不可变的。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/authorization-resources/role-binding-v1.md:60]:
```suggestion
  如果无法解析 roleRef，则 Authorizer 必定返回一个错误。这个字段是不可变更的。
```

**Review by ydFu — APPROVED** [review]
/lgtm
/approve

**Pull request merged**